generator/operator.go

```diff
@@ -1,14 +1,200 @@
 package main
 
+import (
+	"net/http"
+
+	"github.com/hashicorp/nomad/api"
+)
+
 func (v *v1api) getOperatorPaths() []*apiPath {
-	// tags := []string{"Operator"}
+	tags := []string{"Operator"}
 
 	return []*apiPath{
 		//s.mux.HandleFunc("/v1/operator/license", s.wrap(s.LicenseRequest))
+		// NOTE: there's no license endpoint
```

Comment:
Good call. Yes, it probably should be.

Reply:
just added a TODO note for this for the time being